schema.sql

```diff
@@ -10,4 +10,33 @@
 
```

Comment:
![image](https://github.com/IsmailChangezi/vet-clinic/assets/93039359/803167e8-7e57-4a36-a637-62d3eebca276)

**[OPTIONAL]**

- [ ] It would be nice if you add the following insertion queries into your data.sql file :+1: to follow the project requirements :blush:

Reply:
The only problem with this one was that I had added the following commands in the schema.sql but I have moved them to data.sql again.

Reply:
Actually, you still have some insertions queries in the schema.sql file. Kindly put them in the data.sql file because the data.sql file is responsible for containing all the insertion queries :+1: and queries.sql file is responsible for having data retrieval queries :blush: